avocado/core/parser.py

```diff
@@ -46,6 +51,32 @@ def _get_option_tuples(self, option_string):
         return []
 
 
+class StdoutAlreadyClaimedError(Exception):
+
+    """
+    Signals that other player already claimed exclusive rights on STDOUT
+    """
+
+
+class FileOrStdoutAction(argparse.Action):
```

Comment:
Btw how about simply a function `parser.claim_stdout(option)` which would raise exception if already claimed? That's IMO all we need.